README.md

```diff
@@ -164,24 +190,26 @@ Pros of creating a new tool:
   It would be hard to do that while preserving compatibility in jpm.
 * Creating SDK-based add-ons was overly complicated. With web extensions you no
   longer need to convert your source into legacy artifacts and you won't need
-  boostrapping scripts.
+  bootstrapping scripts.
 * There are superior features in Firefox now for developing extensions such
   as [loading](https://blog.mozilla.org/addons/2015/12/23/loading-temporary-add-ons/)
-  from source code instead of a packaged XPI. It will be
-  easier to reimagine a new tool around these work flows rather than
-  adjust jpm's existing work flows.
+  from source code instead of a packaged XPI. It will be easier to reimagine a new tool around these workflows rather than adjusting jpm's existing workflows.
 * jpm's functional tests are slow, brittle and hard to run. There are flaky
   time-outs and we've run out of *low hanging fruit* fixes at this point.
-* Most of jpm's code was not designed to be unit testable which makes it hard to
+* Most of the jpm's code was not designed to be unit testable which makes it hard to
   maintain and refactor.
 * jpm's code was written in ES5 which is cumbersome after coming from the ES6
-  Firefox code base or from most other languages with modern conveniences
+  Firefox codebase or from most other languages with modern conveniences
   (Python, Ruby, etc).
 * Some core functionality of jpm can be extracted and re-used in the new tool.
 
 Cons of creating a new tool:
 
-* Firefox extension developers will have to interrupt and re-arrange their work flows.
+* Firefox extension developers will have to interrupt and re-arrange their workflows.
 * Developers of existing add-ons will need to port to web extensions sooner rather than later.
 * The web-ext tool will require some ramp-up time for scaffolding.
 * The community of jpm contributors will need to shift focus to web-ext.
+
+---
+
+<img src="https://avatars2.githubusercontent.com/u/131524?s=200&v=4" width="50"></img>
```

Comment:
Is this helpful? The repo is already under the mozilla org which I think is enough. I am hesitant to include this URL because I can't tell if it's stable or not.